README.md

```diff
@@ -318,13 +305,6 @@ You can then search using text as usual. Both text and image fields will be sear
 
```

Comment:
For the lines above - because we are only dealing with a single tensor field - the above text won't make as much sense

Reply:
Are you suggesting to remove the sentence `Both text and image fields will be searched:`?

Reply:
yep - because it is no longer true